docs/acp/user/develop-an-app/develop-an-app.md

```diff
@@ -106,7 +125,7 @@ Before your app can be listed in the App Store Catalog, you need to add the foll
 
 The manifest file is the most important one for the app. It contains data that will be displayed in the App Store Catalog. You can use the [manifest code snippet](/docs/acp/user/develop-an-app/code-snippets/manifest-json-file.html) and update it to your needs. Add the manifest file to `config/app/manifest/en_US.json` of your app.
 
-Manifest files must have the local name as the filename, for example, `en_US.json`, and should be placed inside the `config/app/manifest` directory.
+Manifest files must have the locale name as the filename, for example, `en_US.json`, and should be placed inside the `config/app/manifest` directory.
```

Comment:
There is a validation command for this (and the other manifest-related files that are required). Please check if it can be used to validate the format and if it can already be done at this stage of the setup.

Reply:
This command `vendor/bin/acp app:validate` for now unavailable. We will add this when prepare template for PBC in some branch